app/resources/views/pages/edits.txt

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Remove this edits.txt file.  git records what you change.  You don't need to log the changes additionally in a file.